src/main/java/com/github/shyiko/mysql/binlog/io/ByteArrayInputStream.java

```diff
@@ -26,8 +26,8 @@
 public class ByteArrayInputStream extends InputStream {
 
     private InputStream inputStream;
-    private Integer peek;
-    private Integer pos, markPosition;
+    private int peek = -1;
```

Comment:
these changes are all from upstream?

Reply:
looks like it https://github.com/osheroff/mysql-binlog-connector-java/blob/825f21f574e17bfab864eb88818f1611bdf7f37a/src/main/java/com/github/shyiko/mysql/binlog/io/ByteArrayInputStream.java#L29C27-L29C27